frontend/src/data/actions.js

```diff
@@ -17,12 +22,23 @@ export const singleRequest = {
   error: () => ({
     type: types.ERROR,
   }),
-  addUser: ({ name }) => ({
-    type: types.ADDUSER,
-    payload: { name },
+  createUser: ({ email, password }) => ({
+    type: types.CREATEUSER,
+    payload: { email, password },
   }),
-  downloadImage: ({ imageURL }) => ({
-    type: types.DOWNLOADIMAGE,
-    payload: { imageURL },
+  updateEmail: ({ email }) => ({
```

Comment:
Actually sorry one more thing. I think instead of having all the actions in singleRequest, we should create another constant for sign in/create user. So for createUser it should be:

`
export const createUser = {
  request: ...
  success: ...
  error: ...
}
`